src/geo/geojson.js

```diff
@@ -0,0 +1,6 @@
+export default ({ lat, lon }) => {
```

Comment:
is this util needed? this seems extraneous (`util.turf.point` is the same thing)